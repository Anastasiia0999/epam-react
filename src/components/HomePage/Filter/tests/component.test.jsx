import React from 'react';
import { mount} from 'enzyme';

import {Filter} from "../index";

describe('<Filter />', () =>{
    let wrapper;
    let instance;

    const props = {
        fields:{
            "placeholder" : "Search by name"
        },
        filterByNameDispatch: jest.fn(),
        filer: jest.fn()
    };

    beforeEach(() => {
        wrapper = mount(<Filter {...props} />);
        instance = wrapper.instance();
    });

    it('should render correctly', () =>{
        expect(wrapper.html()).toMatchSnapshot();
    });


});


