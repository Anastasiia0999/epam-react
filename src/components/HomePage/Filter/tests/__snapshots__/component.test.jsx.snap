// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Filter /> should render correctly 1`] = `"<div class=\\"md-form col-12 search-field\\"><input class=\\"form-control col-5 search-input my-2\\" type=\\"text\\" aria-label=\\"Search\\"></div>"`;
